src/test/java/com/github/fzakaria/ascii85/Ascii85Test.java

```diff
@@ -105,4 +105,25 @@ public void overflowTest() {
         assertNotNull(Ascii85.decode(encodedString));
     }
 
+    @Test
+    public void testZeroStringDecodes() {
+        testZeroStringDecode(530);
+        testZeroStringDecode(531);
+        testZeroStringDecode(532);
+        testZeroStringDecode(533);
+        testZeroStringDecode(534);
+    }
+
+    public void testZeroStringDecode(int length) {
+        byte[] data = new byte[length];
+        for (int c = 0; c < length; c++) data[c] = 0;
+        try {
+            String encoded = Ascii85.encode(data);
+            byte[] decoded = Ascii85.decode(encoded);
+            assertEquals(length, decoded.length);
+        } catch (Exception e) {
```

Comment:
This try-catch block is totally redundant. In fact, it replaces the explanation provided by JUnit with a generic "assertion failed" stack trace.